src/sampleworks/eval/occupancy_utils.py

```
@@ -1,82 +1,89 @@
+    >>> occupancy_to_str(A=0.5, B=0.3, C=0.2)
+    '0.5occA_0.3occB_0.2occC'
     """
-    if abs(occ_a - 1.0) < 1e-6:
-        return "1.0occAconf"
-    elif abs(occ_a) < 1e-6:
-        return "1.0occBconf"
-    else:
-        occ_b = round(1.0 - occ_a, 2)
-        return f"{occ_a}occAconf_{occ_b}occBconf"
+    parts = []
+    for label in sorted(altloc_occupancies, key=lambda l: str(l).upper()):
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)